FastSimulation/Tracking/python/iterativeTk_cff.py

```diff
@@ -0,0 +1,34 @@
+##############################
+# FastSim equivalent of RecoTracker/IterativeTracking/python/iterativeTk_cff.py
+##############################
+
+import FWCore.ParameterSet.Config as cms
+
+from FastSimulation.Tracking.InitialStep_cff import *
+from FastSimulation.Tracking.DetachedTripletStep_cff import *
+from FastSimulation.Tracking.LowPtTripletStep_cff import *
+from FastSimulation.Tracking.PixelPairStep_cff import *
+from FastSimulation.Tracking.MixedTripletStep_cff import *
+from FastSimulation.Tracking.PixelLessStep_cff import *
+from FastSimulation.Tracking.TobTecStep_cff import *
+# the following loads a dummy empty track collection
+# such that FastSim can import earlyGeneralTracks_cfi from full tracking
+# todo: actual implementation of JetCore iteration  
+from FastSimulation.Tracking.JetCoreRegionalStep_cff import *
+
+import RecoTracker.FinalTrackSelectors.earlyGeneralTracks_cfi
+# todo, import MuonSeededStep_cff, preDuplicateMergingGeneralTracks_cfi, MergeTrackCollections_cff, ConversionStep_cff
+
+generalTracksBeforeMixing = RecoTracker.FinalTrackSelectors.earlyGeneralTracks_cfi.earlyGeneralTracks.clone()
+
+iterTracking = cms.Sequence(
+    InitialStep
+    +DetachedTripletStep
+    +LowPtTripletStep
+    +PixelPairStep
+    +MixedTripletStep
+    +PixelLessStep
+    +TobTecStep
+    +JetCoreRegionalStep
```

Comment:
It's there such that we can import the cfg for generalTracks from FullSim.
Let's keep it there for now. 
When rebasing, I'll see whether I can remove it without crashing GeneralTracks